src/pages/oncall-user/oncall-user.ts

```diff
@@ -46,6 +46,14 @@ export class OncallUserPage {
     });
   }
 
+  getSmsUrl(smsNumber) {
```

Comment:
Same comments as OncallMe for sanitizing the SMS. Also, consider merging these pages to avoid code duplication like this.

Reply:
merged